api/provision/start-provision-job.ts

```diff
@@ -54,7 +54,7 @@ export async function baseHandler(event: ILambdaEvent): Promise<APIGatewayProxyR
  * @param request - provisioning request from SNOW
  * @returns - transformed request to send to the targeted CSP
  */
-function transformProvisionJob(request: ProvisionRequest): CspInvocation {
+export function transformProvisionJob(request: ProvisionRequest): CspInvocation {
```

Comment:
Suggest renaming this.  I think `Job` has connotations that are misleading, as though this involves the provisioning jobs queue which is further downstream in our sequence.

```suggestion
export function translateProvisionRequest(request: ProvisionRequest): CspInvocation {
```

Reply:
Yeah, that makes sense, removing `Job` makes it clearer.